README.rst

```diff
@@ -1,92 +1,126 @@
 Installation
 ============
 
-Dependencies can be installed using setup.py in the same way as synapse: see synapse/README.rst.  For instance::
+Installing the system dependencies
+----------------------------------
 
-    sudo apt-get install build-essential python2.7-dev libffi-dev \
+To install Sydent's dependencies on a Debian-based system, run::
+
+    sudo apt-get install build-essential python3-dev libffi-dev \
                          sqlite3 libssl-dev python-virtualenv libxslt1-dev
 
-    virtualenv -p python2.7 ~/.sydent
+Creating the virtualenv
+-----------------------
+
+To create the virtual environment in which Sydent will run, run::
+
+    virtualenv -p python3 ~/.sydent
     source ~/.sydent/bin/activate
     pip install --upgrade pip
     pip install --upgrade setuptools
-    pip install https://github.com/matrix-org/sydent/tarball/master
 
-Having installed dependencies, you can run sydent using::
 
-    python -m sydent.sydent
+Installing using `pip`
```

Comment:
```suggestion
Installing the latest Sydent release from PyPI
```

To make clear that the sections differentiate between "installing a release" and "installing from the currently checked-out codebase.